extensions/iceberg/build.gradle

```diff
@@ -38,16 +38,18 @@ dependencies {
     implementation "org.apache.iceberg:iceberg-core:${icebergVersion}"
     implementation "org.apache.iceberg:iceberg-core:${icebergVersion}:tests"
     implementation "org.apache.iceberg:iceberg-aws:${icebergVersion}"
+    // The following lines can be uncommented when we enable support for Azure and GCP
     // implementation "org.apache.iceberg:iceberg-azure:${icebergVersion}"
     // implementation "org.apache.iceberg:iceberg-gcp:${icebergVersion}"
     implementation "org.apache.iceberg:iceberg-bundled-guava:${icebergVersion}"
 
     implementation "org.apache.hadoop:hadoop-common:${hadoopVersion}"
     implementation "org.apache.hadoop:hadoop-hdfs-client:${hadoopVersion}"
 
-    runtimeOnly "org.apache.iceberg:iceberg-aws-bundle:${icebergVersion}"
+    // The following lines can be uncommented when we enable support for Azure and GCP
+    // runtimeOnly "org.apache.iceberg:iceberg-aws-bundle:${icebergVersion}"
     // runtimeOnly "org.apache.iceberg:iceberg-azure-bundle:${icebergVersion}"
-    // runtimeOnly "org.apache.iceberg:iceberg-gcp-bundle:${icebergVersion}"
+    runtimeOnly "org.apache.iceberg:iceberg-gcp-bundle:${icebergVersion}"
```

Comment:
Looks like this got enabled and aws-bundle got commented out, but your comment says it should not be. Can you please verify?